addons/mouse-pos/addon.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Mouse position",
+  "description": "Displays your mouse position in the editor.",
```

Comment:
Maybe you should be more clear on when the x/y position appears, and where? Something like `Displays the  x and y coordinates for the mouse position next to the green flag, when on editor mode.`